src/sentry/notifications/utils/__init__.py

```diff
@@ -127,10 +129,27 @@ def summarize_issues(
     return rv
 
 
-def get_link(group: Group, environment: str | None) -> str:
+def get_group_settings_link(
+    group: Group,
+    environment: str | None,
+    rule_details: Sequence[NotificationRuleDetails] | None = None,
+    alert_timestamp: int | None = None,
```

Comment:
When do we use this parameter? Is it just in tests? If that's the case, you could use `freeze_time` in your test to lock it to a specific timestamp

Reply:
Yeah, its used only in acceptance tests. See: https://github.com/getsentry/sentry/blob/89d57ef7363eaacc907e4db1f31768946be9090a/tests/fixtures/emails/alert.txt

It's hard-coded into file, I'm not sure freezing it would work?

Reply:
You could hardcode to a specific time stamp that matches the fixture. Not a huge deal either way.